src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.cpo.data;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "case_payment_orders")
+public class CasePaymentOrderEntity {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    @CreationTimestamp
+    private LocalDateTime createdTimestamp;
+
+    @Column(nullable = false)
+    private LocalDateTime effectiveFrom;
+
+    @Column(length = 16, nullable = false)
+    private String caseId;
```

Comment:
on DataStore caseId is a 'Long'. Any reason for having it as String here?

Reply:
The confluence doc specified `text` - I can refactor it to Long if that is what is required here

Reply:
Changed to Long as per discussion